examples/02-javascript-basics/array_manipulation_library.js

```diff
@@ -9,93 +11,93 @@ Join - join the array elements into a string given a parameter
 */
 
 const $ = {
-  filter: function () {},
-  map: function () {},
-  reduce: function () {},
-  head: function () {},
-  tail: function () {},
-  join: function () {}
+	filter: function (arr, fx) {},
```

Comment:
@paulomilitante I think you also need to remove `(arr, fx)`

Reply:
@adimasuhid 
Ah, I missed this one. I have removed the parameters now.